src/Compilers/Core/Portable/InternalUtilities/FatalError.cs

```diff
@@ -134,7 +134,7 @@ public static bool ReportAndPropagateUnlessCanceled(Exception exception, ErrorSe
         /// <see cref="CancellationToken.IsCancellationRequested"/> set if cancellation is expected.</param>
         /// <returns><see langword="false"/> to avoid catching the exception.</returns>
         [DebuggerHidden]
-        public static bool ReportAndPropagateUnlessCanceled(Exception exception, CancellationToken contextCancellationToken, ErrorSeverity severity)
+        public static bool ReportAndPropagateUnlessCanceled(Exception exception, ErrorSeverity severity, CancellationToken contextCancellationToken)
```

Comment:
Did 'Change Signature' make this easy?

Reply:
It did! Although sometimes it did nothing at all, and my effort to debug that kept resulting in the debugger breaking. :smile: